# Copyright (C) 2021 by NavInfo Europe B.V. The Netherlands - All rights reserved
# Information classification: Confidential
# This content is protected by international copyright laws.
# Reproduction and distribution is prohibited without written permission.
ouster_initializer:
    ros__parameters:
        hostname: "192.168.2.221" # lidar IP address
        udp_dest_host: "192.168.2.45" # This pc ip
        lidar_mode: "1024x20" # possible resolution and rate: either 512x10, 512x20, 1024x10, 1024x20, or 2048x10
        timestamp_mode: "TIME_FROM_PTP_1588" # method used to timestamp measurements: TIME_FROM_INTERNAL_OSC, TIME_FROM_SYNC_PULSE_IN, TIME_FROM_PTP_1588
        lidar_port: 7502 # port to which the sensor should send lidar data
        imu_port: 7503 # port to which the sensor should send imu data
        timeout_sec: 30
